department-of-veterans-affairs/abd-vro
# Exclude RRD from builds and GitHub actions

## What was the problem?

The VRO platform is no longer supporting the RRD product, yet our builds were dependent on the RRD product to complete.

Associated tickets or Slack threads:
https://github.com/department-of-veterans-affairs/abd-vro/issues/1509

## How does this fix it?[^1]
Excludes RRD related code and tests from builds and actions (except for `rrd-shared` which exposes objects used by different components of the code).

## How to test this PR
- App tests


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
